src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import jwt from 'jsonwebtoken';
+
+export const authmiddleware = (req, res, next) => {
+  const auth = req.headers.authorisation;
+
+  if (!auth) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  const [, token] = auth.split(' ');
```

Comment:
The `split(' ')` method assumes that the authorization header is in the format 'Bearer token'. Ensure that this format is always used, or add checks to handle different formats.